app/client/src/git/sagas/checkoutBranchSaga.ts

```
@@ -110,12 +112,19 @@ export default function* checkoutBranchSaga(
         }
       }
     }
-  } catch (error) {
-    yield put(
-      gitArtifactActions.checkoutBranchError({
-        ...basePayload,
-        error: error as string,
-      }),
-    );
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)